src/tests/ftest/rebuild/cascading_failures.yaml

```diff
@@ -12,7 +12,7 @@ timeout: 360
 server_config:
   name: daos_server
   servers:
-    targets: 8
+    targets: 1
```

Comment:
Specific reason we moved from 8 targets to 1?

Reply:
> Specific reason we moved from 8 targets to 1?
> 
> ==> The workaround per DAOS-5611.

I checked the comments of DAOS-5611. It looks like @phender have a solution for the problem. It will be better to go ahead with an actual solution instead of a work around.

Reply:
The solution @phender ref to DAOS-2799.
Per comments on DAOS-2799 about this issue:
It is not blocking anything at this point.  The workaround of specifying a small number of targets in test's yaml file is sufficient to get tests to pass in CI.

Reply:
I would say the approach should be to expose bugs rather than satisfying a criteria to get a pass in CI. If 8 targets is exposing some bug, I will be more inclined towards using that.

Reply:
The issue of 8 targets calculation is related to DAOS-2799, which is not related to this rebuild test, we should not let the failure blocking the rebuild test.